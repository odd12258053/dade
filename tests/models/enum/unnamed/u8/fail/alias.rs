use dade::model;
#[model]
enum TestModel {
    Value(
       #[field(alias = "val")]
        u8
    ),
}
fn main() {}
