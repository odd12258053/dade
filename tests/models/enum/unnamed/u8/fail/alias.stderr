error: No support alias term on the unnamed field.
 --> tests/models/enum/unnamed/u8/fail/alias.rs:4:10
  |
4 |       Value(
  |  __________^
5 | |        #[field(alias = "val")]
6 | |         u8
7 | |     ),
  | |_____^
